content/de/docs/glossary/_index.md

```diff
@@ -0,0 +1,167 @@
+- validieren, dass die Chart Datei nicht manipuliert wurde
+- den Inhalt der Chart Metadaten validieren (`Chart.yaml`)
+- schnell vergleichen, dass das Chart mit seinen Herkunftsdaten übereinstimmt
+
+Herkunftsdateien haben die Erweiterungen `.prov` und können von einem Chart
+Verzeichnis Server oder einem anderen HTTP Server ausgeliefert werden.
+
+## Release (Version)
+
+Wenn ein Chart installiert ist, erstellt die Helmbibliothek ein Version (_release_),
+um nachzuvollziehen, was installiert wurde.
+
+Ein einzelnes Chart wird möglicherweise mehrmals in denselben Cluster installiert
+und erstellt unterschiedliche Versionen. Zum Beispiel, ein Chart kann drei
+PostgreSQL Datenbanken mit demselben Kommando  `helm install` dreimal mit
+unterschiedlchen Versionsnamen installieren.
+
+## Release Number (Release Version)
+
+Eine eintelne Version kann mehrmals aktualisiert werden. Ein sequentieller Zähler
+wird benutzt, um die Versionen nachzuvollziehen. Nach einem ersten `helm install`
+hat eine Version die _release number_ 1. Nach jeder Aktualisierung oder jedem
+Zurückrollen wird die Versionsnummer erhöht.
+
+## Rollback (Zurückrollen)
+
+Eine Version kann zu einem neuen Chart oder einer neuen Konfiguration
+aktualisiert werden. Wenn die Versionshistorie gespeichert ist,
+kann eine Version zu einer vorherigen auch zurückgerollt werden (_rolles back_).
+Das passiert mit dem `helm rollback` Kommando.
+
+Wichtig zu wissen, dass eine zurückgerollte Version eine neue Versionsnummer
+generiert.
+
+| Operation  | Release Number                                       |
+|------------|------------------------------------------------------|
+| install    | release 1                                            |
+| upgrade    | release 2                                            |
+| upgrade    | release 3                                            |
+| rollback 1 | release 4 (but running the same config as release 1) |
+
+Die obige Tabelle zeigt, wie Versionsnummern bei jeder Aktion wie Installieren,
+Aktualisieren und Zurückrollen erhöht werden.
+
+## Helm Library (Helm Bibltiothek oder SDK)
+
+Die Helm Bibliothek (Library oder SDK) referenziert zum Go Code, der direkt
+mit dem Kubernetes API Server zum Installieren, Aktualisieren, Abfragen und
+Löschen von Kubernetes Resourcen interagiert. Sie kann zum Import in ein
+Projekt benutzt werden, um Helm in einer Bibliothek anstatt des
+Kommandozeilenwerkzeugs zu verwenden.
+
+## Repository (Verzeichnis, Repo, Chart Repository)
+
+Helm Charts sind gespeichert in dedizierten HTTP Servern namens
+_chart repositories_ (_Verzeichnis_, _Repositories_ oder einfach _repos_).
+
+Ein Chart Repository Server ist ein einfacher HTTP Server, der eine index.yaml
+Datei ausliefern kann, die eine Anzahl Charts beschreibt und Informationen
+bereitstellt, wo das Chart herunterzuladen ist.
+(Viele Chart Repositories liefern die Charts zusammen mit der `index.yam` Datei
```

Comment:
index.yaml

Reply:
I merged without this fix since it's a minor point - go ahead and fix this in your next PR. Thanks!